etc/csp_whitelist.xml

```diff
@@ -23,7 +23,9 @@
                 <value id="extend-prod-images" type="host">shipping-offers-static-images-bucket-prod.s3.amazonaws.com</value>
                 <value id="extend-dev-images" type="host">shipping-offers-static-images-bucket-dev.s3.amazonaws.com</value>
                 <value id="extend-demo-images" type="host">shipping-offers-static-images-bucket-demo.s3.amazonaws.com</value>
+                <value id="extend-static-assets" type="host">helloextend-static-assets.s3.amazonaws.com</value>
+                <value id="fallback-image" type="host">https://s3.amazonaws.com/ccspersistenceprod-contentstaticassets04b201d4-1apqb8dyegznm/offers/learnMoreModal-default-1654273334107-learnMoreModal.backgroundImageUrl_Generic_WomanwithBox2.jpg</value>
```

Comment:
Did we confirm this value will be static?

Reply:
Yes we did!